### ISSUE TITLE
Add C. elegans muscle cell sample

### ISSUE THREAD

**Issue by tarelli** [openworm/org.geppetto#267]
A NeuroML conversion of the model for the [C. elegans muscle cell](https://github.com/openworm/muscle_model) is being tracked [here](https://github.com/openworm/OpenWorm/issues/169). The model is based on the original work from [Boyle and Cohen 2008](http://www.sciencedirect.com/science/article/pii/S0303264708001408).  
A mechanical model of the muscle cell is also available in Sibernetic and can be run using the Sibernetic Geppetto porting.
We are now in a position that we can start moving towards integration of the two.
The first step would be creating a Geppetto simulation in which the two aspects are simulated independently, this issue wants to track this first task.
- [x] Create a Geppetto simulation sample with one entity and one electrical aspect whose model is the NeuroML model of the muscle cell
- [x] Create a Geppetto simulation sample with one entity and one mechanical aspect whose model is an SPH model of the muscle cell (@skhayrulin can help with this)
- [ ] Add sample to Geppetto

**Comment by tarelli** [comment]
@aribrich sorry for the delayed reply but I was on holiday :) I think @pgleeson is the best person to answer this, Padraig can you help?

**Comment by pgleeson** [comment]
@aribrich sorry too for the delay... Compilation of jNeuroML from source on Windows has worked in the past, but doesn't get tested regularly so may be broken at the moment (https://github.com/NeuroML/jNeuroML/issues/18). 

A better option for you though is getting the stable jNeuroML jar file using subversion as outlined here: https://github.com/NeuroML/jNeuroML/blob/master/README.md.

Then, just set the environment variable JNML_HOME to the the directory containing jnml.bat, add this location too to your PATH variable, and jnml should run from anywhere.

Doing an *svn update* in the install dir will make sure you always have the latest development version of jNeuroML.

**Comment by aribrich** [comment]
@pgleeson thanks for the update tip, I must have had an out of date version. That being said, I think the version of jnml.git I have still points to strange places. I had to add the ..jNeuroML/target directory to my path as well since that is where the .jar files are and they could not be found otherwise. I can probably go through it and get it to point to the proper location, but that is the file I have unchanged. I also had to add JLEMS to my main path as well. It is working now though, so thanks for the help!

**Comment by tarelli** [comment]
Blocked by #51 and #52

**Comment by adrianq** [comment]
@tarelli no more blocking issues... :)

**Comment by tarelli** [comment]
This was added in Release 0.2.1.

**Comment by slarson** [comment]
@tarelli ok one question about this though.  Earlier in the comment stream we are talking about Dan's mechanical aspect of the muscle model, and what has been implemented now is the electrical aspect.  Is there another card to track the mechanical side sample getting incorporated still?

**Comment by tarelli** [comment]
@slarson you can load the muscle cell with this:
```
Simulation.loadFromContent('<?xml version="1.0" encoding="UTF-8"?> <tns:simulation xmlns:tns="http://www.openworm.org/simulationSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="../../main/resources/schema/simulationSchema.xsd"> <tns:entity> <tns:id>sample</tns:id> <tns:aspect> <tns:id>fluid</tns:id> <tns:simulator> <tns:simulatorId>sphSimulator</tns:simulatorId> </tns:simulator> <tns:model> <tns:modelInterpreterId>sphModelInterpreter</tns:modelInterpreterId> <tns:modelURL>https://www.dropbox.com/s/2675axg7lkiksfa/sphModel_elastic_contractible_7220_oneMuscle.xml?dl=1</tns:modelURL> </tns:model> </tns:aspect> </tns:entity> <tns:script>https://raw.github.com/openworm/org.geppetto.samples/master/SPH/LiquidSmall/TEST_LIQUID_SMALL_SCRIPT.js</tns:script> </tns:simulation>')
```

I haven't added this as a separate sample as it's not more interesting than the elastic sample we already have. As for the integration of the two that's tracked by [this](https://github.com/openworm/org.geppetto/issues/307).

**Comment by slarson** [comment]
Got it -- will subscribe to 307.  Thanks!
ᐧ
